src/routers.ts

```diff
@@ -0,0 +1,7 @@
+export enum Route {
+  ROOT = '/',
```

Comment:
Let's keep this within the route file, they won't be used anywhere and not  change to be reused .